### ISSUE TITLE
RawFileSystem.ReleaseDir is missing cancel parameter

### ISSUE THREAD

**Issue by andrewchambers** [hanwen/go-fuse#447]
This seems like an accident - though might be annoying/mostly pointless to fix.

**Comment by hanwen** [comment]
yeah, you're right, but the error is the other way around. Release() probably shouldn't have a cancel channel, because we can't return an error value to the kernel.
